loom/common/core/performance.cpp

```diff
@@ -233,13 +239,10 @@ void LoomProfiler::reset()
 
 LoomProfilerRoot::LoomProfilerRoot(const char *name)
 {
-    for (LoomProfilerRoot *walk = sRootList; walk; walk = walk->mNextRoot)
-    {
-        if (!strcmp(walk->mName, name))
-        {
-            lmAssert(false, "Duplicate profile name: %s", name);
-        }
-    }
+    LookupType* lookup = static_cast<LookupType*>(sRootLookup);
+    utHashedString key(name);
```

Comment:
Not after merging with that string hash fix that also checks for collisions by comparing the exact strings.